homeassistant/components/abode/__init__.py

```diff
@@ -100,73 +84,79 @@
 class AbodeSystem:
     """Abode System class."""
 
-    def __init__(self, username, password, cache, name, polling, exclude, lights):
+    def __init__(self, abode, polling):
         """Initialize the system."""
 
-        self.abode = abodepy.Abode(
-            username,
-            password,
-            auto_login=True,
-            get_devices=True,
-            get_automations=True,
-            cache_path=cache,
-        )
-        self.name = name
+        self.abode = abode
         self.polling = polling
-        self.exclude = exclude
-        self.lights = lights
         self.devices = []
 
-    def is_excluded(self, device):
-        """Check if a device is configured to be excluded."""
-        return device.device_id in self.exclude
 
-    def is_automation_excluded(self, automation):
-        """Check if an automation is configured to be excluded."""
-        return automation.automation_id in self.exclude
+async def async_setup(hass, config):
+    """Set up Abode platform."""
```

Comment:
This is the component module. We could write "integration" which is the whole package.

Reply:
Sounds good. I was just referencing what the Hue platform had for their doc strings. :) Just pushed a commit with it changed to "integration".